From 5ab888762fd612f18a2bf23d0185f78b21b04f37 Mon Sep 17 00:00:00 2001
From: Jingdong Lu <jingdong.lu@windriver.com>
Date: Thu, 27 Oct 2011 09:42:29 +0800
Subject: [PATCH 54/60] Enable omap44xx_aess_hwmod, omap44xx_mcpdm_hwmod

Enable omap44xx_aess_hwmod, omap44xx_mcpdm_hwmod
Modify name of these modules
Modify omap44xx_aess_addrs

Signed-off-by: Jingdong Lu <jingdong.lu@windriver.com>
---
 arch/arm/mach-omap2/omap_hwmod_44xx_data.c |   16 ++++++++--------
 1 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm/mach-omap2/omap_hwmod_44xx_data.c b/arch/arm/mach-omap2/omap_hwmod_44xx_data.c
index 63e7c41..f7c99ec 100644
--- a/arch/arm/mach-omap2/omap_hwmod_44xx_data.c
+++ b/arch/arm/mach-omap2/omap_hwmod_44xx_data.c
@@ -665,7 +665,7 @@ static struct omap_hwmod_class_sysconfig omap44xx_aess_sysc = {
 };
 
 static struct omap_hwmod_class omap44xx_aess_hwmod_class = {
-	.name	= "aess",
+	.name	= "omap-aess-audio",
 	.sysc	= &omap44xx_aess_sysc,
 };
 
@@ -692,8 +692,8 @@ static struct omap_hwmod_ocp_if *omap44xx_aess_masters[] = {
 
 static struct omap_hwmod_addr_space omap44xx_aess_addrs[] = {
 	{
-		.pa_start	= 0x401f1000,
-		.pa_end		= 0x401f13ff,
+		.pa_start	= 0x49000000, 
+		.pa_end		= 0x491f11ff,
 		.flags		= ADDR_TYPE_RT
 	},
 };
@@ -711,7 +711,7 @@ static struct omap_hwmod_ocp_if omap44xx_l4_abe__aess = {
 static struct omap_hwmod_addr_space omap44xx_aess_dma_addrs[] = {
 	{
 		.pa_start	= 0x490f1000,
-		.pa_end		= 0x490f13ff,
+		.pa_end		= 0x490f11ff,
 		.flags		= ADDR_TYPE_RT
 	},
 };
@@ -733,7 +733,7 @@ static struct omap_hwmod_ocp_if *omap44xx_aess_slaves[] = {
 };
 
 static struct omap_hwmod omap44xx_aess_hwmod = {
-	.name		= "aess",
+	.name		= "omap-aess-audio",
 	.class		= &omap44xx_aess_hwmod_class,
 	.mpu_irqs	= omap44xx_aess_irqs,
 	.mpu_irqs_cnt	= ARRAY_SIZE(omap44xx_aess_irqs),
@@ -3068,7 +3068,7 @@ static struct omap_hwmod_class_sysconfig omap44xx_mcpdm_sysc = {
 };
 
 static struct omap_hwmod_class omap44xx_mcpdm_hwmod_class = {
-	.name	= "mcpdm",
+	.name	= "omap-mcpdm-dai",
 	.sysc	= &omap44xx_mcpdm_sysc,
 };
 
@@ -3126,7 +3126,7 @@ static struct omap_hwmod_ocp_if *omap44xx_mcpdm_slaves[] = {
 };
 
 static struct omap_hwmod omap44xx_mcpdm_hwmod = {
-	.name		= "mcpdm",
+	.name		= "omap-mcpdm-dai",
 	.class		= &omap44xx_mcpdm_hwmod_class,
 	.mpu_irqs	= omap44xx_mcpdm_irqs,
 	.mpu_irqs_cnt	= ARRAY_SIZE(omap44xx_mcpdm_irqs),
@@ -5051,7 +5051,7 @@ static __initdata struct omap_hwmod *omap44xx_hwmods[] = {
 	&omap44xx_mpu_private_hwmod,
 
 	/* aess class */
-/*	&omap44xx_aess_hwmod, */
+	&omap44xx_aess_hwmod, 
 
 	/* bandgap class */
 	&omap44xx_bandgap_hwmod,
-- 
1.7.4.1

